processor/otel/consumer.go

```diff
@@ -65,9 +66,29 @@ const (
 
 // Consumer transforms open-telemetry data to be compatible with elastic APM data
 type Consumer struct {
+	stats consumerStats
+
 	Reporter publish.Reporter
 }
 
+// ConsumerStats holds statistics about data consumption.
+type ConsumerStats struct {
+	// UnsupportedMetricsDropped records the number of unsupported metrics
+	// that have been dropped by the consumer.
+	UnsupportedMetricsDropped int64
+}
+
+type consumerStats struct {
```

Comment:
I am confused about this, why both `consumerStats` and `ConsumerStats` are needed?

Reply:
`consumerStats` might change at any time, `ConsumerStats` is a snapshot. I'll add a comment.